### PR TITLE
Temporary responses.coffee anti-spam update

### PR DESCRIPTION
**Pull request by Chigg**
With the intention of slowing down some of the most common responses, I think this a good temporary fix. If people really want, they can comeback and re-add them. But the key purpose is to de-clutter in an attempt to remove spam. I kept some of the rarer ones I've never seen called, and indiscriminately removed the others.

### CODE REVIEW

File: scripts/responses.coffee [hunk 1]
```diff
@@ -82,83 +82,83 @@ module.exports = (robot) ->
...
 
-  robot.hear /(^|\s+)stevie/i, (msg) ->
-    msg.send "@steven"
+ # robot.hear /(^|\s+)stevie/i, (msg) ->
+  #  msg.send "@steven"
 
-  robot.hear /real *man/i, (msg) ->
-    msg.send "I think you mean a Real Hacker, genders yo"
+ # robot.hear /real *man/i, (msg) ->
+  #  msg.send "I think you mean a Real Hacker, genders yo"
```
Comment (arrdem) [inline scripts/responses.coffee:92]: In honor of Clay and Taylor I protest to this

File: scripts/responses.coffee [hunk 2]
```diff
@@ -82,83 +82,83 @@ module.exports = (robot) ->
...
 
-  robot.hear /^\s*texas!?\s*$/i, (msg) ->
-    msg.send "FIGHT"
+  #robot.hear /^\s*texas!?\s*$/i, (msg) ->
+    #msg.send "FIGHT"
 
-  robot.hear /^\s*ou!?\s*$/i, (msg) ->
-    msg.send "SUCKS"
+  #robot.hear /^\s*ou!?\s*$/i, (msg) ->
+    #msg.send "SUCKS"
```
Comment (arrdem) [inline scripts/responses.coffee:136]: Come on. What school do we go to. These two stay.

File: scripts/responses.coffee [hunk 3]
```diff
@@ -82,83 +82,83 @@ module.exports = (robot) ->
...
 
-  robot.respond /rule (5|five)$/i, (msg) ->
-    msg.send "5. Reid if you want to meet girls you need to do things that girls do"
+  #robot.respond /rule (5|five)$/i, (msg) ->
+    #msg.send "5. Reid if you want to meet girls you need to do things that girls do"
 
-  robot.hear /rule (34|thirty four)$/i, (msg) ->
-    msg.send "34. If it is on the internet, there is porn of it"
+  #robot.hear /rule (34|thirty four)$/i, (msg) ->
+    #msg.send "34. If it is on the internet, there is porn of it"
```
Comment (arrdem) [inline scripts/responses.coffee:161]: Eh. These aren't that bad, and they're pretty relevant when used.

File: scripts/responses.coffee (hunk 1 above)
Comment (rainiera) [inline scripts/responses.coffee:92]: +1

**Comment by Chigg** [comment]
Like I said, I indiscriminately removed the others. I don't think personal opinion should get in the way of unbiased removal. Otherwise people will ask "Why was mine removed and this one wasn't" etc. If it's not going to be removed it's probably just going to be exploited again, because I know people have made a point to trigger lanbot's responses.

**Pull request closed without merging**